### DISCUSSION TITLE
lebahmovie.com - empty ads placeholders

### DISCUSSION THREAD

**Discussion by Recehan-Slayer** [realodix/AdBlockID#730] [General]
**Jenis iklan** (hapus yang tidak perlu)
Ads Placeholder

**Sample URL**
https://lebahmovie.com/

**Penjelasan Lebih Lanjut**
tempat ads yg tersisa

**Comment by realodix** [comment]
Overlay ya? Mengingat maintain domain movie yang sulit karena gonta-ganti terus dan tidak terlalu mengganggu, Saya rasa ini tidak masalah untuk didiamkan saja. Thank you

**Comment by Recehan-Slayer** [comment]
@realodix klo web lebahganteng ini kyknya ga gnti" domain deh yg ganti" domain itu golonganya indoxx1 ilk21 lk21 ganool dan sejenisnya

**Comment by realodix** [comment]
Mohon untuk dimengerti Gan, lebahmovie.com sama seperti situs indoxx1 dan LK21, mereka adalah situs yang menampilkan film bajakan dan termasuk situs ilegal di Indonesia. Efeknya begitu mereka sudah mulai terkenal, mereka akan tercium oleh pemerintah, lalu mereka akan kucing-kucingan dengan pemerintah dengan berganti-ganti domain.

Kecuali situs-situs nonton legal seperti pada referensi ini, kecil banget kemungkinan ganti domain dengan sangat cepat.
- [https://www.grid.id/read/04196590...avoritmu?page=all](https://www.grid.id/read/041965903/indoxxi-diblokir-7-layanan-streaming-legal-ini-bisa-jadi-alternatif-pilihan-nonton-film-favoritmu?page=all)
- [https://inet.detik.com/cyberlife/d-483495...lm-legal](https://inet.detik.com/cyberlife/d-4834950/indoxxi-ditutup-ini-pilihan-layanan-streaming-film-legal)
- [https://www.tribunnewswiki.com/2019/12/24/indoxxi-...ad-legal](https://www.tribunnewswiki.com/2019/12/24/indoxxi-akan-dihapus-ini-10-situs-lain-untuk-nonton-film-gratis-bisa-streaming-dan-download-legal)

**Comment by Recehan-Slayer** [comment]
ohh ok brarti yg belum gnti" domain visitornya blom bnyak bgt ya

**Comment by realodix** [comment]
> ohh ok brarti yg belum gnti" domain visitornya blom bnyak bgt ya

Visitor banyak / nama mulai naik daun seperti Indoxx1 dan LK21 -> tercium pemerintah, lalu kucing-kucingan deh.

Itu hanya gambaran umum aja sih Gan, karena Saya juga gak tau teknik apa yang digunakan oleh pemerintah agar masuk radar. Makanya karena mereka sering banget kucing-kucingan, Saya bener2 utamain iklan (terutama iklan dewasa) untuk spesifik blok dan spesifik hide.

**Comment by Recehan-Slayer** [comment]
@realodix klo disitus" bokep jg termasuk ga gan kan bnyak juga tuh banner" iklan judi iklan porno dll

**Comment by realodix** [comment]
Bisa Gan, tapi ya gitu, hanya generic filter aja, tanpa spesifik filter.

**Comment by Recehan-Slayer** [comment]
@realodix ada iklan judinya
https://lebahmovie.com/eps/virgin-river-season-1-episode-10/

**Comment by realodix** [comment]
> ada iklan judinya
> https://lebahmovie.com/eps/virgin-river-season-1-episode-10/

Sorry, di bagian mananya Gan? Di Gue bersih

<details>
  <summary>Screencapture</summary>
  
  ![screencapture-lebahmovie-eps-virgin-river-season-1-episode-10-2020-01-06-17_13_16](https://user-images.githubusercontent.com/1314456/71811677-eb164b80-30a7-11ea-985e-a8313d89ec7d.png)
</details>

**Comment by Recehan-Slayer** [comment]
@realodix ato munkin gegara ublocknya 
![z5](https://user-images.githubusercontent.com/9379770/71812171-151c3d80-30a9-11ea-9424-097593baf36e.png)

**Comment by realodix** [comment]
Kalau di Firefox gimana Gan? Dengan konfigurasi filter yang sama.

Screenshoot di atas Saya pakai Chrome + uBlock Origin dgn konfigurasi
- All Built-in, kecuali xxperimental
- EasyList​​​​​
- EasyPrivacy
- Malware domains
- AdBlockID

**Comment by Recehan-Slayer** [comment]
@realodix klo di firefox bersih dgn filter yg sama yg gue gunain di chrome klo penyebabnya ubo laporin gan

**Comment by Recehan-Slayer** [comment]
@realodix nih filter yg gunain klo di firefox bersih tp di chrome kok masi muncul padahal sama smua filter+semua extension yg gw pakai sama persis
<details>

uBlock filters​​​​​18,635 used out of 19,324​​​​
uBlock filters – Annoyances​​​​​2,623 used out of 2,834​​​​
uBlock filters – Badware risks​​​​​810 used out of 819​​​​
uBlock filters – Privacy​​​​​159 used out of 175​​​​
uBlock filters – Resource abuse​​​​​97 used out of 116​​​​
uBlock filters – Unbreak​​​​​1,211 used out of 1,286​​​​
Ads (3/4)
Adblock Warning Removal​​​​​657 used out of 740​​​​
AdGuard Base​​​​​30,583 used out of 31,211​​​​
EasyList​​​​​85,529 used out of 86,598​​​​
Privacy (2/3)
AdGuard Tracking Protection​​​​​8,449 used out of 8,508​​​​
EasyPrivacy​​​​​13,137 used out of 17,856​​​​
Malware domains (2/4)
Malware Domain List​​​​​1,105 used out of 1,105​​​​
Malware domains​​​​​26,848 used out of 26,863​​​​
Annoyances (3/6)
AdGuard Annoyances​​​​​23,980 used out of 24,095​​​​
EasyList Cookie​​​​​0 used out of 16,366​​​​
Fanboy’s Annoyance​​​​​53,725 used out of 55,964​​​​
Multipurpose (1/4)
Peter Lowe’s Ad and tracking server list​​​​​3,114 used out of 3,245​​​​
Regions, languages (0/35)
Custom (4/5)
AdBlockID​​​​​10,042 used out of 10,054​​​​
Nano Defender Integration​​​​​2,645 used out of 2,646​​​​
Nano Filters​​​​​344 used out of 344​​​​
Nano Filters - Whitelist​​​​​1 used out of 1​​​

**Comment by realodix** [comment]
Saya juga gak ngerti itu kenapa di Chrome masih muncul?

Menurut SS dari agan, iklannya mengarah ke `https://tgks.dev/lebahmovie`, coba tolong lihat di log deh, ada filter `##[href*="tgks."] > img` gak? Jika sudah update ada keluar filter `##[href*="tgks.dev"] > img` atau `##a[title*="bola tangkas online" i] > img` gak?

**Comment by Recehan-Slayer** [comment]
@realodix 
![Untitled3](https://user-images.githubusercontent.com/9379770/71834060-e1f5a080-30e0-11ea-8eef-e304d5dd478b.png)

**Comment by realodix** [comment]
Anehnya terjadi di Chrome doang.

Atau mungkin berhubungan dengan ini `https://github.com/uBlockOrigin/uBlock-issues/issues/338`

Coba deh matikan beberapa filter

**Comment by Recehan-Slayer** [comment]
ok sprtinya mslhnya dichrome

**Comment by Recehan-Slayer** [comment]
@realodix coba tolong bntuin bro tolong cek pke filter diatas di browser google chrome
gw dah coba matiin 1 per 1 filternya tpi tetep aja msh muncul beberapa iklan yg udah ada difilter seperti tgks.***
kira" sebabnya gara" filter yg mana? atau browsernya?

**Comment by realodix** [comment]
> coba tolong bntuin bro tolong cek pke filter diatas di browser google chrome
> gw dah coba matiin 1 per 1 filternya tpi tetep aja msh muncul beberapa iklan yg udah ada difilter seperti tgks.***
> kira" sebabnya gara" filter yg mana? atau browsernya?

Dari yg Saya pelajari, sepertinya masalahnya muncul bukan karena filter (filter bentrok satu sama lain) deh Bro, contohnya seperti #108, karena muncul hanya di Chrome saja, Firefox tidak.

Ciri-cirinya selalu sama
- Hanya terjadi di Chrome
- Selalu menyisakan 1 banner, padahal menurut uBlock logger sudah dihilangkan.

<details>
  <summary>Screenshot 1</summary>

  ![cats](https://user-images.githubusercontent.com/1314456/71852840-467a2500-310c-11ea-8b09-cbabe9eebeae.jpg)
</details>

---

**Kondisi**

Pada kondisi apa yang sebenarnya dapat mentrigger Saya juga tidak tahu, tapi ada 2 kondisi yang Saya temukan.

- **Diduga uBlock jadi aneh saat** junglah network filters ＋ cosmetic filters hasilnya lebih dari 200k. Pada Screenshot 2 uBlock Origin jadi aneh.

<details>
  <summary>Screenshot 2</summary>

  
  ![cats](https://user-images.githubusercontent.com/1314456/72064529-a3ddc400-330e-11ea-941f-3bfea10b0464.jpg)

</details>


- **Diduga uBlock berjalan sempurna saat** junglah network filters ＋ cosmetic filters hasilnya di bawah 200k. Pada Screenshot 3, jumlahnya adalah 196.327 (filter ngasal, untuk ngepasin angkanya saja) dan uBlock Origin berjalan dengan sempurna.

<details>
  <summary>Screenshot 3</summary>

  ![cats](https://user-images.githubusercontent.com/1314456/72063880-439a5280-330d-11ea-8f4a-a1db67047199.jpg)
</details>

**Comment by realodix** [comment]
**Diduga uBlock jadi aneh saat** cosmetic filters lebih dari 65k. Junmlahnya di bawah 200k, tapi cosmetic filters lebih dari 65k.

<details>
  <summary>Screenshot 4</summary>
  
  ![cats](https://user-images.githubusercontent.com/1314456/72065357-9fb2a600-3310-11ea-9978-01f6d9be71c8.jpg)

</details>

**Comment by Recehan-Slayer** [comment]
@realodix jdi ama google dibatasin gitu

**Comment by realodix** [comment]
> jdi ama google dibatasin gitu

Apa yg sebenarnya terjadi juga Saya tidak tahu Gan, masih bersifat asumsi aja. Masih Saya pelajari.

**Comment by realodix** [comment]
Saya buat issue baru untuk lebahmovie.com khusus utuk iklan, kebeneran ada banner judi yang baru. Jika ada pembahasan lebih lanjut, kita bisa bahas di sana.
